docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -0,0 +1,389 @@
+
+
+***********
+Validation
+***********
+
+1. ``otx eval`` runs evaluation of a trained model on a particular dataset.
+
+Eval function receives test annotation information and model snapshot, trained in previous step.
+Please note, that ``label schema.json`` file should be located in the same folder with model snaphot, as it contains meta information about the dataset .
+
+The default metric measured is F1 measure.
+
+2. The command below evaluates snaphot in ``outputs`` folder on BCCD dataset and saves results to ``outputs/performance`` :
+
+.. code-block::
+
+  (detection) ...$ otx eval otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --test-ann-files BBCD/valid/_annotations.coco.json 
+                            --test-data-roots  BBCD/valid 
+                            --load-weights outputs/weights.pth
+                            --save-performance outputs/performance.json
+  
+
+If ``data.yaml`` was created, the command can be simplified by passing it for a ``--data`` parameter. Note, that this line will run validation on the test set (not validation set):
+
+.. code-block::
+
+  (detection) ...$ otx eval otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --data data.yaml 
+                            --load-weights outputs/weights.pth
+                            --save-performance outputs/performance.json
+
+The validation output will look as following:
+
+.. code-block::
+
+  2022-12-29 01:32:00,505 | INFO : run task done.
+  2022-12-29 01:32:01,215 | INFO : Inference completed
+  2022-12-29 01:32:01,216 | INFO : called evaluate()
+  2022-12-29 01:32:01,527 | INFO : F-measure after evaluation: 0.8315842078960519
+
+
+
+Additionally, you can tune testing parameters such as confidence threshold via comand line. You can read more about template-specific parameters for validation in quick start [#TODO link].
+For example, to increase the confidence treshold to decrease the number of False Positive predictions (there you have prediction, but don't have annotated object for it) update the evaluation comand line as it's shown below. 
+Please note, that by default confidence treshold is detected automatically based on result to maximize final F1 metric. So, to set custom confidence trashold, please disable ``result_based_confidence_threshold`` option.
+
+.. code-block::
+
+  (detection) ...$ otx eval otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --data data.yaml 
+                            --load-weights outputs/weights.pth
+                            params 
+                            --postprocessing.confidence_threshold 0.5
+                            --postprocessing.result_based_confidence_threshold false 
+
+...
+
+2023-01-03 18:55:01,956 | INFO : F-measure after evaluation: 0.6274238227146813
```

Comment:
![Screenshot from 2023-01-06 10-06-04](https://user-images.githubusercontent.com/13793254/210978662-6eab2fa9-67bf-4014-a667-b831a5dcd0be.png)

The format looks wrong.